iroh-dns-server/src/server.rs

```diff
@@ -29,7 +28,7 @@ pub async fn run_with_config_until_ctrl_c(config: Config) -> Result<()> {
         store = store.with_mainline_fallback(bootstrap);
     };
     let server = Server::spawn(config, store, metrics).await?;
-    tokio::signal::ctrl_c().await?;
+    tokio::signal::ctrl_c().await.context("ctrl-c")?;
```

Comment:
These `.context`s aren't super fun. Should we change them to `.e()`?

Reply:
sure